common/metrics/defs.go

```diff
@@ -1565,6 +1567,8 @@ var ScopeDefs = map[ServiceIdx]map[int]scopeDefinition{
 		BlobstoreClientDeleteScope:          {operation: "BlobstoreClientDelete", tags: map[string]string{CadenceRoleTagName: BlobstoreRoleTagValue}},
 		BlobstoreClientDirectoryExistsScope: {operation: "BlobstoreClientDirectoryExists", tags: map[string]string{CadenceRoleTagName: BlobstoreRoleTagValue}},
 
+		DefaultIsolationGroupStateHandlerGetAvailableIsolationGroupsScope: {operation: "DefaultIsolationGroupStateHandlerGetAvailableIsolationGroups"},
```

Comment:
The name is bad. Don't put `DefaultIsolationGroupStateHandler` here. If you want to have this information, create another tag.

Reply:
yeah, good point, I'm blunding around a bit here with naming